pkg/swagger/types.go

```diff
@@ -164,6 +164,7 @@ type Response struct {
 	Schema      *Schema `json:"schema,omitempty"`
 	Headers     Headers `json:"headers,omitempty"`
 	Examples    Example `json:"examples,omitempty"`
+	SystemData  *Schema `json:"systemData,omitempty"`
```

Comment:
what is this for?

Reply:
ARM CI/sign-off required this for all new APIs/API Versions. I think we will need to make change not to log this in any away in our code but overall we need it now